index.html

```diff
@@ -20,7 +20,7 @@
   <script>
   // Scatter Plot
   var scatterSpec = {
-    "data": {"url": "data/cars.json"},
+    "data": {"url": "https://vega.github.io/vega-datasets/data/cars.json"},
```

Comment:
This means it won't work offline?  

Better setup a build script to copy data like we do in Vega-Lite.